core/src/lib/tls_openssl.h

```diff
@@ -50,6 +50,7 @@ class TlsOpenSsl : public Tls {
 
   std::string TlsCipherGetName() const override;
   void SetCipherList(const std::string& cipherlist) override;
+  void SetProtocol(std::string protocol) override;
```

Comment:
every other signature in this domain is `const std::string&`, is there any reason not to keep it consistent?